app/src/util/tools/index.ts

```diff
@@ -0,0 +1,20 @@
+export {
+  calcPoolTokens,
+  calcAddFundingSendAmounts,
+  calcDepositedTokens,
+  calcRemoveFundingSendAmounts,
+  calcDistributionHint,
+  calcInitialFundingSendAmounts,
+  calcAddFundingDepositedAmounts,
+  calcInitialFundingDepositedAmounts,
+  calcSharesBought,
+} from './fpmm/liquidity'
```

Comment:
Can we switch these up to `export * from...` now to make it super clean?

Reply:
ohh yeah good catch dunoo how that slipped!